server/routes/createPlan/checkYourAnswers/checkYourAnswersController.ts

```diff
@@ -42,7 +49,62 @@ export default class CheckYourAnswersController {
 
     try {
       const record = getSessionData(req, ['createPlan', id])
+      const newCiagPlan: CiagPlan = {
```

Comment:
Ideally I would separate this into a different mapper class, but the `record` object currently has no type, which makes it a bit ugly. Perhaps we should revisit this whole class once we're calling our API.

Reply:
Sounds like a plan 👍